src/main/java/com/github/lunatrius/schematica/proxy/ClientProxy.java

```diff
@@ -206,8 +206,8 @@ public static boolean saveCoordinatesFile(Map<String, Map<String, Map<String, In
         }
     }
 
-    public static boolean addCoordinates(String worldServerName, String schematicName, Integer X, Integer Y,
-            Integer Z) {
+    public static boolean addCoordinates(String worldServerName, String schematicName, Integer X, Integer Y, Integer Z,
```

Comment:
As this now also saves the rotation it should be reflected in the method name, "addCoordinatesAndRotation" should work well enough